lib/updatehub/updatehub.c

```diff
@@ -4,9 +4,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <logging/log.h>
+#define LOG_MODULE_NAME updatehub
+#define LOG_LEVEL CONFIG_UPDATEHUB_LOG_LEVEL
 
-LOG_MODULE_REGISTER(updatehub);
+#include <logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 
```

Comment:
This works, but can be written shorter way (up to you if you want to change that).
```
#include <logging/log.h>
LOG_MODULE_REGISTER(updatehub, CONFIG_UPDATEHUB_LOG_LEVEL);
```

Reply:
Sure! I'll change it as you suggested.